src/patternfly/components/ExpandableSection/expandable-section.scss

```diff
@@ -23,16 +23,60 @@
 
   // Content
   --pf-c-expandable-section__content--MarginTop: var(--pf-global--spacer--md);
+  --pf-c-expandable-section__content--MaxWidth: inherit;
+
+  // Large
+  --pf-c-expandable-section--m-display-lg--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-expandable-section--m-display-lg__toggle--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-expandable-section--m-display-lg__toggle--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-expandable-section--m-display-lg__toggle--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-expandable-section--m-display-lg__toggle--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-expandable-section--m-display-lg__content--MarginTop: 0;
+  --pf-c-expandable-section--m-display-lg__content--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-expandable-section--m-display-lg__content--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-expandable-section--m-display-lg__content--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-expandable-section--m-display-lg__content--MaxWidth: var(--pf-global--breakpoint--md);
```

Comment:
I'm not sure that we would want to use a breakpoint var here (we have only used breakpoint vars in `@media` queries). For example, if a user updated the breakpoint value to be something else, would we expect the content width in this component to match? This max-width likely won't match an actual media query breakpoint on the page since the expandable section will likely be in a page section and/or card or something which has its own padding, meaning the viewport media query will take effect before the expandable content max-width (set to the same breakpoint size) is reached.

If it's just supposed to limit the width to improve readability, we have a similar [`max-width` set on the form component](https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/Form/form.scss#L11) (it's `500px`) available as an opt-in as `.pf-m-limit-width`. I wonder if we should do something like that? And if we might benefit from a global variable we use for things like this for consistency.

wdyt @ajacobs21e @mcarrano @srambach?

Reply:
My understanding was that it was for readability. I've moved the `max-width` from the large variant to the `.pf-m-limit-width` modifier with a value of 750px (since that's what is in the spec).